src/ServiceControl.Management.PowerShell/Particular.ServiceControl.Management.format.ps1xml

```diff
@@ -15,8 +15,8 @@
                 <PropertyName>Name</PropertyName>
               </ListItem>
               <ListItem>
-                <Label>TypeName</Label>
-                <PropertyName>TypeName</PropertyName>
+                <Label>DisplayName</Label>
```

Comment:
I think we might want to make some additional changes here. I'm not sure that there's a point in showing the display name? Also, there's some weirdness around "deprecated" things. For example, the learning transport is showing in the output, but as deprecated.